Resources/Locale/en-US/traits/traits.ftl

```diff
@@ -11,7 +11,7 @@ trait-description-Pacifist = You cannot attack or hurt any living beings.
 trait-name-LightweightDrunk = Lightweight Drunk
 trait-description-LightweightDrunk = Alcohol has a stronger effect on you
 
-trait-name-HeavyweightDrunk = Heavyweight Drunk
+trait-name-HeavyweightDrunk = Alcohol Tolerance
 trait-description-HeavyweightDrunk = Alcohols are afraid of you
```

Comment:
```suggestion
trait-description-HeavyweightDrunk = Alcohol is afraid of you
```
I'm in anti-fun mood.

Reply:
Gotta add a period too. The lack of a period triggers me

Reply:
Resolved with [e4375f0](https://github.com/Simple-Station/Einstein-Engines/pull/669/commits/e4375f0f9ab2c841ed9126a3d7aa9ce3b49db026)